urdf2webots/parserURDF.py

```diff
@@ -413,6 +416,51 @@ def getSTLMesh(filename, node):
     return node
 
 
+def getOBJMesh(filename, node):
+    """read obj file."""
+    print('Parsing Mesh: ' + filename)
+    with open(filename, 'r') as file:
+        trimesh = node.geometry.trimesh
+        for line in file:
+            (header, body) = line.split(maxsplit=1)
+            if header == '#':
+                continue
+            elif header == 'f':  # face
+                vertices = body.split()
+                coordIndex = []
+                texCoordIndex = []
+                normalIndex = []
+                for vertex in vertices:
+                    indices = vertex.split('/')
+                    coordIndex.append(int(indices[0]) - 1)  # vertex coordinates
+                    length = len(indices)
+                    if length > 1 and indices[1]:  # texture coordinates
+                        texCoordIndex.append(int(indices[1]) - 1)
+                    if length > 2 and indices[2]:  # normal coordinates
+                        normalIndex.append(int(indices[2]) - 1)
+                trimesh.coordIndex.append(coordIndex)
+                if texCoordIndex:
+                    trimesh.texCoordIndex.append(texCoordIndex)
+                if normalIndex:
+                    trimesh.normalIndex.append(normalIndex)
+            elif header == 'l':  # line, ignored
+                continue
+            elif header == 'o':  # object name, ignored
+                continue
+            elif header == 'v':  # vertex
+                (x, y, z) = body.split()
+                trimesh.coord.append([float(x), float(y), float(z)])
+            elif header == 'vn':  # normals
+                (x, y, z) = body.split()
+                trimesh.normal.append([float(x), float(y), float(z)])
+            elif header == 'vp':  # parameters, ignored
+                continue
+            elif header == 'vt':  # texture coordinate, ignored
```

Comment:
Why do we ignore texture coordinate?

Reply:
I could not test it, so I preferred to not implement it yet. But I can implement it blindly...

Reply:
(the OBJ format is a bit tricky about the number of parameters here, so the implementation is not straightforward)

Reply:
Ok that's fine for me, I was just wondering if it was just a matter of adding the trimesh.texCoord (because from the documetnation of obj file it looked lije they were simply defined as `vt 1.0 0.0`).

Reply:
Not that simple (from [wikipedia](https://en.wikipedia.org/wiki/Wavefront_.obj_file)):
```
# List of texture coordinates, in (u, [,v ,w]) coordinates, these will vary between 0 and 1. v, w are optional and default to 0.
  vt 0.500 1 [0]
```
But doable. I will try to implement it.